app/controllers/team_preferences_controller.rb

```diff
@@ -0,0 +1,18 @@
+class TeamPreferencesController < ApplicationController
+  def create_teams
```

Comment:
move this to app/admin/team_preferences.rb and have it trigger the functions in there